Lesson-7/index.html

```diff
@@ -58,33 +73,35 @@ <h2 class="headline">What do we offer</h2>
 </section>
 
 <section class="Homes_guests_loves">
-<h2 class="heading_homes_guests_loves">Homes guests loves</h2>
-<div class="hotels">
-<div class="hotel_leopold">
-<img class="leopold_img" src="image/Leopold.png" alt="leopold_img">
-<p class="text_hotel_leopold">Hotel Leopold</p>
-<p class="location_leopold">Saint Petersburg, Russia</p>
-</div>
-<div class="apartment_sunshine">
-<img class="sunshine_img" src="image/sunshine.png" alt="sunshine_img">
-<p class="text_apartment_sunshine">Apartment Sunshine</p>
-<p class="location_sunshine">Santa Cruz de Tenerife, Spain</p>
-</div>
-<div class="villa_kunerad">
-<img class="kunerad_img" src="image/kunerad.png" alt="kunerad_img">
-<p class="text_villa_kunerad">Villa Kunerad</p>
-<p class="location_kunerad">Vysokie Tatry, Slowakia</p>
-</div>
-<div class="hostel_friendship">
-<img class="frendship_img" src="image/frendship.png" alt="frendship_img">
-<p class="text_hostel_friendship">Hostel Friendship</p>
-<p class="location_frendship">Berlin, Germany</p>
-</div>
-<button>></button>
-</div>
-</section>
+    <h2 class="heading_homes_guests_loves">Homes guests loves</h2>
+    <div class="hotels">
+    <div class="hotel_leopold">
+    <img class="leopold_img" src="image/Leopold.png" alt="leopold_img">
+    <p class="text_hotel_leopold">Hotel Leopold</p>
+    <p class="location_leopold">Saint Petersburg, Russia</p>
+    </div>
+    <div class="apartment_sunshine">
+    <img class="sunshine_img" src="image/sunshine.png" alt="sunshine_img">
+    <p class="text_apartment_sunshine">Apartment Sunshine</p>
+    <p class="location_sunshine">Santa Cruz de Tenerife, Spain</p>
+    </div>
+    <div class="villa_kunerad">
+    <img class="kunerad_img" src="image/kunerad.png" alt="kunerad_img">
+    <p class="text_villa_kunerad">Villa Kunerad</p>
+    <p class="location_kunerad">Vysokie Tatry, Slowakia</p>
+    </div>
+    <div class="hostel_friendship">
+    <img class="frendship_img" src="image/frendship.png" alt="frendship_img">
+    <p class="text_hostel_friendship">Hostel Friendship</p>
+    <p class="location_frendship">Berlin, Germany</p>
+    </div>
+    <button class="slider-arrows">></button>
+    </div>
+    </div>
+    </section>
+
 
-<section class="last">
+<footer class="last_block">
```

Comment:
The footer doesn't need to be given classes. it can be accessed directly to add styles by `footer {}`